src/main/java/com/lohika/morning/ecs/domain/unsubscribe/UnsubscribeService.java

```diff
@@ -0,0 +1,41 @@
+package com.lohika.morning.ecs.domain.unsubscribe;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+import org.springframework.util.StringUtils;
+
+import java.util.List;
+
+import static java.util.Arrays.stream;
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class UnsubscribeService {
+  @Autowired
+  private UnsubscribeRepository unsubscribeRepository;
+
+  public List<Unsubscribe> findAll() {
+    return unsubscribeRepository.findAll();
+  }
+
+  public List<Unsubscribe> filterByEmail(String email) {
+    return unsubscribeRepository.filterAllByEmail(email);
+  }
+
+  public void add(String string) {
+    List<Unsubscribe> emails = stream(string.split("\\s+"))
```

Comment:
On the UI, you wrote "Comma separated emails", but here you are splitting by space. So if I write "asc@example.com,zxc@example.com" it will be saved as one email.